test-suite/src/test/scala/org/scalajs/testsuite/jsinterop/ExportsTest.scala

```diff
@@ -561,6 +561,10 @@ object ExportsTest extends JasmineTest {
       expect(obj.witness).toEqual("witness")
     }
 
+    it("should allow exporting an object as toString - #1723") {
+      expect(js.Dynamic.global.applyDynamic("toString")().x).toEqual(2)
```

Comment:
I don't like this test, because it actually pollutes the global object of the test suite with a `toString()` method that doesn't even return a string. I suggest writing this as a pos test in `compiler/test`.